src/main/java/marquez/core/mappers/CoreNamespaceToApiNamespaceMapper.java

```diff
@@ -2,15 +2,15 @@
 
 import static java.util.Objects.requireNonNull;
 
-import marquez.api.models.Namespace;
+import marquez.api.models.NamespaceResponse;
 
 // TODO: Move to marquez.api.mappers pgk
 // TODO: Rename class to NamespaceMapper
 public class CoreNamespaceToApiNamespaceMapper
-    extends Mapper<marquez.core.models.Namespace, Namespace> {
-  public Namespace map(marquez.core.models.Namespace namespace) {
+    extends Mapper<marquez.core.models.Namespace, NamespaceResponse> {
+  public NamespaceResponse map(marquez.core.models.Namespace namespace) {
```

Comment:
With the `NamespaceResponse` class name change, you don't need the full `marquez.core.models.` path for `Namespace`.